### PR TITLE
fix @jwt and @jwtClaim entries in the directives overview table

### PR DESCRIPTION
**Pull request by rsill-neo4j**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-150.surge.sh

Updated pages:
- [Directives](https://neo4j-docs-graphql-150.surge.sh/graphql/5/type-definitions/directives)

**Review by lidiazuin — APPROVED** [review]
LGTM!

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 1]
```diff
@@ -43,8 +43,11 @@ of any required fields that is passed as arguments to the custom resolver.
 | xref::/type-definitions/directives/autogeneration.adoc#type-definitions-autogeneration-id[`@id`]
 | Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
 
-| xref::/authentication-and-authorization/configuration.adoc#authentication-and-authorization-jwt[`@jwtClaim`]
-| Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
+| xref::/authentication-and-authorization/configuration.adoc#authentication-and-authorization-jwt[`@jwt`]
+| Configure the JWT authentication filter to include an additional JWT claim.
+
+| xref::/authentication-and-authorization/configuration.adoc#_nested_claims[`@jwtClaim`]
+| Configure the JWT authentication filter to include an additional nested JWT claim.
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/type-definitions/directives/index.adoc]:
```suggestion
| Use in combination with `@jwt`. Configure the JWT authentication filter to include an additional nested JWT claim.
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-150.surge.sh

Updated pages:
- [Directives](https://neo4j-docs-graphql-150.surge.sh/graphql/5/type-definitions/directives)

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 2]
```diff
@@ -43,8 +43,11 @@ of any required fields that is passed as arguments to the custom resolver.
 | xref::/type-definitions/directives/autogeneration.adoc#type-definitions-autogeneration-id[`@id`]
 | Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
 
-| xref::/authentication-and-authorization/configuration.adoc#authentication-and-authorization-jwt[`@jwtClaim`]
-| Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
+| xref::/authentication-and-authorization/configuration.adoc#authentication-and-authorization-jwt[`@jwt`]
+| Configure the JWT authentication filter to include an additional JWT claim.
```
Comment (darrellwarde) [inline modules/ROOT/pages/type-definitions/directives/index.adoc]:
```suggestion
| Configure the JWT authentication and authorization filters to include additional JWT claims.
```

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 3]
```diff
@@ -43,8 +43,11 @@ of any required fields that is passed as arguments to the custom resolver.
...
+
+| xref::/authentication-and-authorization/configuration.adoc#_nested_claims[`@jwtClaim`]
+| Use in combination with `@jwt`. Configure the JWT authentication filter to include an additional nested JWT claim.
```
Comment (darrellwarde) [inline modules/ROOT/pages/type-definitions/directives/index.adoc]:
```suggestion
| Use in combination with `@jwt`. Configure the JWT authentication and authorization filters to include an additional JWT claim which is either nested or using special characters not supported by GraphQL.
```

**Review by darrellwarde — CHANGES_REQUESTED** [review]
This looks good, I've just made some suggestions to maybe slightly improve the wording here. 🙂

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-150.surge.sh

Updated pages:
- [Directives](https://neo4j-docs-graphql-150.surge.sh/graphql/5/type-definitions/directives)

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 4]
```diff
@@ -43,8 +43,11 @@ of any required fields that is passed as arguments to the custom resolver.
 | xref::/type-definitions/directives/autogeneration.adoc#type-definitions-autogeneration-id[`@id`]
 | Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
 
-| xref::/authentication-and-authorization/configuration.adoc#authentication-and-authorization-jwt[`@jwtClaim`]
-| Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
+| xref::/authentication-and-authorization/configuration.adoc#authentication-and-authorization-jwt[`@jwt`]
+| Configure the JWT authentication and authorization filters to include additional JWT claims.
+
+| xref::/authentication-and-authorization/configuration.adoc#_nested_claims[`@jwtClaim`]
+| Use in combination with `@jwt`. Configure the JWT authentication and authorization filters to include an additional JWT claim which is either nested or using special characters not supported by GraphQL.
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/type-definitions/directives/index.adoc]:
```suggestion
| Used in combination with `@jwt`. Configures the JWT authentication and authorization filters to include an additional JWT claim which is either nested or using special characters not supported by GraphQL.
```

File: modules/ROOT/pages/type-definitions/directives/index.adoc [hunk 5]
```diff
@@ -43,8 +43,11 @@ of any required fields that is passed as arguments to the custom resolver.
 | xref::/type-definitions/directives/autogeneration.adoc#type-definitions-autogeneration-id[`@id`]
 | Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
 
-| xref::/authentication-and-authorization/configuration.adoc#authentication-and-authorization-jwt[`@jwtClaim`]
-| Marks a field as the unique ID for an object type, and allows for autogeneration of IDs.
+| xref::/authentication-and-authorization/configuration.adoc#authentication-and-authorization-jwt[`@jwt`]
+| Configure the JWT authentication and authorization filters to include additional JWT claims.
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/type-definitions/directives/index.adoc]:
```suggestion
| Configures the JWT authentication and authorization filters to include additional JWT claims.
```

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Comment by rsill-neo4j** [comment]
the changes suggested are incorporated in this PR: https://github.com/neo4j/docs-graphql/pull/140

**Pull request closed without merging**